airflow-core/src/airflow/ui/public/i18n/locales/en/common.json

```diff
@@ -102,6 +102,7 @@
     "dags": "Dags",
     "docs": "Docs",
     "home": "Home",
+    "legacyFabViews": "Legacy FAB Views",
```

Comment:
Text is a bit tight in the menu, can you change it to "Legacy Views"?
```suggestion
    "legacyFabViews": "Legacy Views",
```

Reply:
See also screenshow above...

Reply:
Yes, thanks! Done